ASUWebPlatforms/webspark-ci
# WS2-2090: Add new Hover Cards block

### Description

This PR adds a new Hover Cards Layout Builder block.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-2090)

### QA Steps

- Pull the branch locally into your existing env (do this instead of using the PR multidev so you can test the update hooks).
- Run drush updb. Update hook webspark_update_10011 should run without error.
- Create or edit an existing page that can use the Layout Builder.
- Add a new block, and select “Create content block”.
- Verify there is one new option in the list: Hover Cards.
- Add a Hover Cards block. Ensure all available content options work as expected (data entered appear on the front end, required form fields are validated properly, anchor menu links work as expected, spacing works as expected, etc)
- In your local env, create a backup of your database as needed.
- Wipe the database, and perform a new installation.
- Create a new Basic Page.
- Ensure you can add a block, and the Hover Cards options appear and work as expected.

### Additional Notes

When you are testing this, be aware some items may be removed, such as the Tooltip, if I get word that it is not needed for this block.

### Checklist


### Verified in browsers
